nixos/modules/services/networking/wireguard.nix

```diff
@@ -160,6 +160,14 @@ let
         interval of 25 seconds; however, most users will not need this.'';
       };
 
+      table = mkOption {
+        default = "main";
+        type = types.str;
+        description = ''The kernel routing table to add this peer's associated
```

Comment:
Okay that's fine. It'll be a pain in the ass if somebody wants all the peers on the same table and there are a lot of peers, but I guess it's just copy and paste for that. So okie dokes.

> So what if multiple peers have the same CIDR range configured as AllowedIPs? 

This can't happen. Watch what happens when you try:

![image](https://user-images.githubusercontent.com/10643/29009529-547af10c-7b24-11e7-8b80-3304f8c77e6e.png)